packages/block-library/src/index.js

```diff
@@ -194,6 +195,7 @@ const getAllBlocks = () => {
 		templatePart,
 		avatar,
 		postTitle,
+		parentPostTitle,
```

Comment:
Hi @nani-samireddy, Here I come up with an idea. Can we extend the functionality of postTitle without creating a complete block `parentPostTitle`? Like add a toggle option to existing `postTitle` that determines if postTitle will show postTitle OR parent postTitle.